monorepo-bench/dune

```diff
@@ -0,0 +1,2111 @@
+(executable
+ (public_name bench)
+ (modules bench)
+ (libraries unix))
+
+(library
```

Comment:
Why do we need this mega library? It's not enough to build everything individually?

Reply:
The monorepo is made up of many packages and some of them contain libraries that don't build. I've made a list of the libraries that don't build and used them to construct the list of libraries in this file.

Reply:
What do you mean don't build? As in fail to build, or aren't being requested to build at all?

Reply:
I mean that they fail to build in a monorepo setting.

Reply:
I see. So building this mega library is supposed to fail as well then?

Reply:
No, the list of libraries in this file are just the ones that build in a monorepo setting.